src/scenes/UserGalleryPage/UserGalleryCollections.tsx

```diff
@@ -69,20 +69,22 @@ function UserGalleryCollections({ galleryRef, queryRef, mobileLayout }: Props) {
 
   const nonNullCollections = removeNullValues(collections);
 
-  const cache = useRef(
-    new CellMeasurerCache({
-      fixedWidth: true,
-      minHeight: 100,
-    })
+  const [cache] = useState(
+    () =>
+      new CellMeasurerCache({
+        fixedWidth: true,
+        minHeight: 100,
+      })
   );
+
```

Comment:
what's the reason for converting this to state?

Reply:
refs should not be used inside of render functions, and `useMemo` does not guarantee the same value every render.